validate/setup-guides/aws/createEC2-2.md

```diff
@@ -0,0 +1,167 @@
+---
```

Comment:
It seems like there's a lot of duplicate text on this page. Can we link to it instead and just include the new content?